code/game/machinery/_machinery.dm

```diff
@@ -326,10 +326,11 @@
  * Will update the machine icon and any user interfaces currently open.
  * Arguments:
  * * drop - Boolean. Whether to drop any stored items in the machine. Does not include components.
+ * * density - Boolean. Whether to make the object dense when it's open.
  */
-/obj/machinery/proc/open_machine(drop = TRUE)
+/obj/machinery/proc/open_machine(drop = TRUE, density = FALSE)
```

Comment:
`density` is a byond-level variable, so this is unfortunately a bit confusing. can you change it to

```suggestion
/obj/machinery/proc/open_machine(drop = TRUE, density_to_set = FALSE)
```

I got confused thinking you were somehow changing the value of the byond-level variable in the arguments of a proc. that might be unreasonable but i got really flummoxed.

Reply:
Gotcha, that makes sense. Should I also adjust all the child overrides to pass in a parameter with this new name? Or is that unnecessary?